master/networking/workloads-outside-cluster.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Configure workload access outside the cluster
+description: Describes how to enable and disable internet access for your workloads.
```

Comment:
I suggest ditching the "Describes how to" preamble on those which have it. WDYT?

More to the point "How to enable and disable internet access for your workloads"

Reply:
Agree.